queue_model_enumerables_v1.py

```diff
@@ -26,8 +26,8 @@
 EDNARefImages = namedtuple('EDNARefImages', ['FOUR', 'TWO', 'ONE', 'NONE'])
 EDNA_NUM_REF_IMAGES = EDNARefImages(0, 1, 2, 3)
 
-CentringMethod = namedtuple('CentringMethod', ['MANUAL', 'LOOP', 'FULLY_AUTOMATIC', "NO"])
-CENTRING_METHOD = CentringMethod(0, 1, 2, 3)
+CentringMethod = namedtuple('CentringMethod', ['NO', 'MANUAL', 'LOOP', 'FULLY_AUTOMATIC', 'XRAY'])
+CENTRING_METHOD = CentringMethod(0, 1, 2, 3, 4)
```

Comment:
Hi @mguijarr ,

You are absolutely right. I will move 'NO' to the end of enumerator.